gulpfile.js

```diff
@@ -6,7 +6,8 @@ build(gulp, {
     js: {
         files: {
             main: {
-                srcPath: 'fozzie.js'
+                srcPath: 'index.js',
```

Comment:
My bad here but on second thoughts you could actually remove this line  as index.js is the default :)